CMakeLists.txt

```diff
@@ -22,12 +22,26 @@ option(BUILD_UMESIMD  "Build UME::SIMD library from source")
 option(BUILD_VC       "Build Vc library from source")
 
 include(CompilerSetup)
+include(GNUInstallDirs)
 
 if (CUDA)
   include(CUDA)
 endif()
 
 add_library(VecCore INTERFACE)
+target_include_directories(VecCore INTERFACE
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+
+# This supports projects using VecCore either as builtin or via find_package
+# A direct ALIAS target is not used because we *do not* want to export
+# the VecCore target. If we do this when it links to a backend, that dependency
+# is exported, and that is artificial. VecCore had no direct dependencies, with
+# configure/compile time choice by the client choosing this. We therefore construct
+# the imported target the same way here as is done in VecCoreConfig.cmake.
+# It is GLOBAL as it will always be created in a subdirectory of any including project
+add_library(VecCore::VecCore IMPORTED INTERFACE GLOBAL)
+target_include_directories(VecCore::VecCore INTERFACE ${PROJECT_SOURCE_DIR}/include)
```

Comment:
This particular target should probably go in the project that wants to vendor VecCore instead of in VecCore itself. I don't really want to encourage vendoring. The reason I have builtins for Vc and UME::SIMD is for testing only, so I don't have to figure out how to install Vc and UME::SIMD in each different CI. The builtins are not meant be used when installing VecCore otherwise.